### PR TITLE
fix:minor style changes

### PR DESCRIPTION
**Pull request by ahmaxed**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: client/src/components/layouts/global.css [hunk 1]
```diff
@@ -273,6 +273,18 @@ code {
...
+.has-error .help-block,
+.has-error .control-label,
+.has-error .radio,
+.has-error .checkbox,
+.has-error .radio-inline,
+.has-error .checkbox-inline,
+.has-error.radio label,
+.has-error.checkbox label,
+.has-error.radio-inline label,
+.has-error.checkbox-inline labe {
```
Comment (scissorsneedfoodtoo) [inline client/src/components/layouts/global.css]: `.has-error.checkbox-inline label {`

File: client/src/pages/donate.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ export class DonatePage extends Component {
               <div className='text-center'>
                 <PoweredByStripe />
                 <Spacer />
-                <Button onClick={this.toggleOtherOptions}>
+                <Button class='btn btn-link' onClick={this.toggleOtherOptions}>
                   {`${
                     showOtherOptions ? 'Hide' : 'Show'
                   } other ways to donate.`}
```
Comment (scissorsneedfoodtoo) [inline client/src/pages/donate.js]: This is a minor thing I noticed that's outside of the scope of this PR. Just wondering if we should remove the period from this button so it better matches the other buttons on this page:

`} other ways to donate`}`

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
Checked this out on GitPod and the styles LGTM. Just found one line in `global.css` that needs to be addressed.

File: client/src/pages/donate.js (hunk 2 above)
Comment (ahmadabdolsaheb) [reply client/src/pages/donate.js]: @scissorsneedfoodtoo although this is a button, it will look like a link.

Comment (scissorsneedfoodtoo) [reply client/src/pages/donate.js]: Sorry, should have been more clear and made the suggestion on line 118. 

What I mean is that we should probably remove the period from the text "Hide/Show other ways to donate" so it better matches the other buttons or links on the page. This seems to be the only button or link with a period on this page and in the settings.

**Comment by scissorsneedfoodtoo** [comment]
Everything LGTM

**Pull request merged**